### ISSUE TITLE
如何训练自己的数据集呢？

### ISSUE THREAD

**Issue by DomineeringDragon** [cure-lab/SCINet#43]
在下菜鸟一枚。之前在informer模型上运行了自己的数据集，现在打算在SCINet上也如法炮制一番。但是遇到了如下的报错：
![23Q2VGVLOO49QA{F9S Z~TD](https://user-images.githubusercontent.com/55223251/169483471-e7409c49-049d-4b8b-9638-a1c3f340ecb7.png)
我是这样处理的：将run_ETTh.py文件中的data_paser添加我的数据集，对应的dataloader是Dataset_Custom，并在ETTH_data_loader.py文件中添加了配置。加载应该是可以的，但是返回了报错。Etth数据集加载的代码结构和informer模型代码的很像，但在informer模型的代码中就没有报错。请问是为什么呢？

**Comment by bwb5165** [comment]
exp_ETTh.py
![image](https://user-images.githubusercontent.com/60076753/169895953-4df8340b-ae53-4b05-ae7b-86532a7a7552.png)

line 28:
7 -> 129

I had a similar error. I found that while using a custom dataset, I needed to manually change the number of feature columns in exp_ETTh.py to the correct number for my dataset.

This should fix your issue.

**Comment by VEWOXIC** [comment]
Yes, we recommend to use the ETT dataloader for training your own dataset. All you need is to change the input dimension as your dataset. The ETT dataloader support both the univariant and multi-variant task.
